bundles/org.openhab.binding.sensorcommunity/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -76,12 +76,12 @@
 	<channel-type id="pm25-channel">
 		<item-type>Number:Density</item-type>
 		<label>Particulate Matter category 2.5</label>
-		<state pattern="%.1f %unit%" readOnly="true"/>
+		<state pattern="%.1f µg/m³" readOnly="true"/>
```

Comment:
Why do you want to hardcode a unit here ?
Is it a different unit from what the binding is using when setting the channel state ?

Reply:
If no unit is set on the item, the default unit (Kilogram per cubic meter) is used internally. And if "%unit%" is set in the state pattern, the value will be displayed in UI in kilogram per cubic meter.
Is it the problem you try to fix ?

Reply:
I believe it is better to set an appropriate unit in state pattern when the default unit is certainly inappropriate for the channel. @jlaur : do you agree ?

Reply:
Yes, I agree. I'm now thinking I should do the same for Miele:
https://github.com/openhab/openhab-addons/blob/57b265574d021f46dd6d8d9ebc28ebbbc2674483/bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/channeltypes.xml#L239-L248

When I introduced this, the unit from the state update would be used, and this is given in liters by the binding. However, since 4.0 this will now be defaulted to m³ which certainly doesn't make sense for measuring water consumption for a washing machine.

Reply:
So this change is good. I just hope the unit is a valid unit. I assumed @weymann tested that the value is displayed converted in the expected unit.